Methods voice change/copy edit/add sidra details (#242)
AlexsLemonade/OpenPBTA-manuscript: content/06.methods.md

PNOC [@url:https://www.pnoc.us/] is an international consortium dedicated to bringing new therapies to children and young adults with brain tumors.
We also included blood and tumor biospecimens from newly-diagnosed diffuse intrinsic pontine glioma (DIPG) patients as part of the PNOC003 clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258].

All CBTN data can be download from the Gabriella Miller Kids First Data Resource Center [@url:https://kidsfirstdrc.org].

The CBTN-generated cell lines from either fresh tumor tissue obtained directly from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing medium (cat# 12648010, Gibco).
We dissociated tumor tissue using enzymatic method with papain as described [@doi:10.1101/656587].
Briefly, we washed tissue with HBSS (cat# 14175095, Gibco), and we minced and incubated the tissue with activated papain solution (cat# LS003124, SciQuest) for up to 45 minutes.

> The papain was inactivated using ovomucoid solution (cat# 542000, SciQuest), tissue was briefly treated with DNase (cat#  10104159001, Roche), and passed through the 100μm cell strainer (cat# 542000, Greiner Bio-One).

Reviewer: Woops, missed one - 
```suggestion
The papain was inactivated using ovomucoid solution (cat# 542000, SciQuest), tissue was briefly treated with DNase (cat#  10104159001, Roche), and passed through the 100μm cell strainer (cat# 542000, Greiner Bio-One).
```
```suggestion
We used ovomucoid solution (cat# 542000, SciQuest) to inactivate tge papain, and we briefly treated tissue with DNase (cat#  10104159001, Roche) and passed it through the 100μm cell strainer (cat# 542000, Greiner Bio-One).
```

For cultures utilizing the fetal bovine serum (FBS), a minimum density of 3×10<sup>5</sup> cells/mL were plated in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza), and 0.2% Normocin (cat# ant-nr-2, Invivogen).
For serum-free media conditions, cells were plated at minimum density of 1×10<sup>6</sup> cells/mL in DMEM/F12 medium supplemented with 1% GlutaMAX, 1X B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/mL basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/mL heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture, and 0.2% Normocin.

#### Nucleic acids extraction and library preparation

##### PNOC samples

The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using a DNA/RNA AllPrep Kit (Qiagen, #80204).
All RNA used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNA.

> For library preparation, 500 ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel  (panel) sequencing.

Author: same as below (sent out and performed by the companies)

> RNA library preparation was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Roche, #KK8201) using Agilent's SureSelect Human All Exon V5 backbone with custom probes.

Author: hmm, technically this would have been sent out to a company to do, so maybe leave as is?

Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).

> Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).

Author: leaving passive

> DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).

Author: this may also be tough as we did not perform extractions in house (our biorespository core did this and they are not currently authors).... but should they be? this probably goes back decades....

> DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).

Author: leaving passive

> Library preparation and sequencing was performed by the NantHealth Sequencing Center, BGI at CHOP, or the Genomic Clinical Core at Sidra Medical and Research Center.

Author: updating in next commit

> Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# 08098107702, Roche); tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# 07962304001, Roche).

Author: not we, so will leave passive

#### Data generation

NantHealth and Sidra performed 2x150 bp WGS on paired tumor (~60X) and constitutive (~30X) DNA samples on an Illumina X/400.
BGI at CHOP performed 2x100 bp WGS sequenced at 60X depth for both tumor and normal samples.
NantHealth performed ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 200M.
BGI at CHOP performed poly-A or ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 100M.
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed paired tumor (~200X) and constitutive whole exome sequencing (WXS) or targeted DNA panel (panel) and poly-A selected RNA-Seq (~200M reads) for  PNOC tumor samples.

> The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.

Author: keep as passive, performed elsewhere

Lastly, for normal/germline input, we ran the GATK HaplotypeCaller [@doi:10.1101/201178] submodule on the recalibrated BAM to generate a genomic variant call format (GVCF) file.
This file is used as the basis for germline calling, described in the **SNV calling for B-allele Frequency (BAF) generation** section.
References were obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket with a general description of references here [@url:https://s3.amazonaws.com/broad-references/broad-references-readme.html].

#### Quality Control of Sequencing Data

To confirm sample matches and remove mis-matched samples from the dataset, we performed NGSCheckmate [@doi:10.1093/nar/gkx193] on matched tumor/normal CRAM files. 
Briefly, we processed CRAMs using BCFtools to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters [@url:https://github.com/parklab/NGSCheckMate]. 
The resulting VCFs were used to run NGSCheckmate following [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) found in the D3b GitHub repository.

> Per author recommendations, we used <= 0.61 as a correlation coefficient cutoff at sequencing depths > 10 to predict mis-matched samples.

Author: NGSCheckmate, added

GATK v4.0.12.0 was used for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.
This single-sample workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found athttps://s3.amazonaws.com/broad-references/broad-references-readme.html.

#### Somatic Mutation Calling

##### SNV and indel calling

For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.
We used only Strelka2, Mutect2 and Lancet to analyze WXS samples from TCGA.

> TCGA samples were captured using various WXS target capture kits and all the BED files were downloaded from [`GDC portal`](https://api.gdc.cancer.gov/files).

Author: this was done by the TCGA project, not us, so will keep passive

> TCGA samples were captured using various WXS target capture kits and all the BED files were downloaded from [`GDC portal`](https://api.gdc.cancer.gov/files).

Author: updated next commit

Both tissue and cell pellets processes included a CHCl3 extraction and were run on the QIACube automated platform (Qiagen) using the AllPrep DNA/RNA/miRNA Universal kit (cat# 80224, Qiagen).
Blood was thawed and treated with RNase A (cat#, 19101, Qiagen); 0.4-1ml was processed using the Qiagen QIAsymphony automated platform (Qiagen) using the QIAsymphony DSP DNA Midi Kit (cat# 937255, Qiagen).
DNA and RNA quantity and quality was assessed by PerkinElmer DropletQuant UV-VIS spectrophotometer (PerkinElmer) and an Agilent 4200 TapeStation (Agilent, USA) for RIN and DIN (RNA Integrity Number and DNA Integrity Number, respectively).

> Active version

Reviewer: :) 
```suggestion
```

Author: whoops

> Active version

Reviewer: :) 
```suggestion
```

The NantHealth Sequencing Center, BGI at CHOP, or the Genomic Clinical Core at Sidra Medical and Research Center, performed library preparation and sequencing.
Briefly, DNA sequencing libraries were prepared for tumor and matched-normal DNA using the KAPA HyperPrep kit (cat# 08098107702, Roche); tumor RNA-Seq libraries were prepared using KAPA Stranded RNA-Seq with RiboErase kit (cat# 07962304001, Roche).

#### Data generation

NantHealth and Sidra performed 2x150 bp WGS on paired tumor (~60X) and constitutive (~30X) DNA samples on an Illumina X/400.
BGI at CHOP performed 2x100 bp WGS sequenced at 60X depth for both tumor and normal samples.
NantHealth performed ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 200M.
BGI at CHOP performed poly-A or ribosomal-depleted whole transcriptome stranded RNA-Seq to an average depth of 100M.
The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed paired tumor (~200X) and constitutive whole exome sequencing (WXS) or targeted DNA panel (panel) and poly-A selected RNA-Seq (~200M reads) for PNOC tumor samples.

> The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.

Reviewer: Just needs a comma 
```suggestion
The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.
```
```suggestion
The panel tumor sample was sequenced to 470X, and the normal panel sample was sequenced to 308X.
```

> The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.

Reviewer: Just needs a comma 
```suggestion
The panel tumor sample was sequenced to 470X and the normal panel sample was sequenced to 308X.
```
```suggestion
The panel tumor sample was sequenced to 470X, and the normal panel sample was sequenced to 308X.
```

#### Quality Control of Sequencing Data

To confirm sample matches and remove mis-matched samples from the dataset, we performed NGSCheckmate [@doi:10.1093/nar/gkx193] on matched tumor/normal CRAM files. 
Briefly, we processed CRAMs using BCFtools to filter and call 20k common single nucleotide polymorphisms (SNPs) using default parameters [@url:https://github.com/parklab/NGSCheckMate]. 
We used the resulting VCFs to run NGSCheckmate following [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
Per NGSCheckmate author recommendations, we used <= 0.61 as a correlation coefficient cutoff at sequencing depths > 10 to predict mis-matched samples.
For RNA-Seq, we determined read strandedness by running the [`infer_experiment.py` script](http://rseqc.sourceforge.net/#infer-experiment-py) on the first 200k mapped reads.
If calculated strandedness did not match strandedness information received from the sequencing center, samples were removed from analysis.

> We required that at least 60% of RNA-Seq reads mapped to the human reference before samples were removed from analysis.

Reviewer: ```suggestion
We required that at least 60% of RNA-Seq reads mapped to the human reference before samples were removed from analysis.
```
```suggestion
We required that at least 60% of RNA-Seq reads mapped to the human reference before removing samples from analysis.
```

> We required that at least 60% of RNA-Seq reads mapped to the human reference before samples were removed from analysis.

Reviewer: ```suggestion
We required that at least 60% of RNA-Seq reads mapped to the human reference before samples were removed from analysis.
```
```suggestion
We required that at least 60% of RNA-Seq reads mapped to the human reference before removing samples from analysis.
```

#### Germline Variant Calling

##### SNP calling for B-allele Frequency (BAF) generation

We performed germline haplotype calls using the [GATK Joint Genotyping Workflow](https://software.broadinstitute.org/gatk/best-practices/workflow?id=11145) on individual GVCFs from the normal sample alignment workflow.
Using only SNPs, we applied the [GATK generic hard filter suggestions](https://gatkforums.broadinstitute.org/gatk/discussion/2806/howto-apply-hard-filters-to-a-call-set) to the VCF, with an additional requirement of 10 reads minimum depth per SNP.
We used the filtered VCF as input to Control-FREEC and CNVkit (below) to generate B-allele frequency (BAF) files.

> GATK v4.0.12.0 was used for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.

Reviewer: Note to either use the `v` or not. I removed, but can be added back in to all the versions.

```suggestion
We used GATK 4.0.12.0 for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this application.
```

> GATK v4.0.12.0 was used for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this purpose.

Reviewer: Note to either use the `v` or not.

```suggestion
We used GATK 4.0.12.0 for all steps except `VariantFiltration`, which used 3.8.0 because as of GATK 4.0.12.0, this tool was beta and known to be unreliable for this application.
```

This single-sample workflow is available in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows).
References can be obtained from the [Broad Genome References on AWS](https://s3.console.aws.amazon.com/s3/buckets/broad-references/hg38/v0/) bucket, and a general description of references can be found at https://s3.amazonaws.com/broad-references/broad-references-readme.html.

#### Somatic Mutation Calling

##### SNV and indel calling

For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].

> VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.

Reviewer: ```suggestion
We did not retain VarDict-only calls because ~ 39M calls with low VAF were uniquely called, suggesting they may have been false positives.
```

> VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.

Reviewer: ```suggestion
We did not retain VarDict-only calls because ~ 39M calls with low VAF were uniquely called, suggesting they may have been false positives.
```

We used only Strelka2, Mutect2 and Lancet to analyze WXS samples from TCGA.

> TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).

Reviewer: comma

```suggestion
TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).
```
```suggestion
TCGA samples were captured using various WXS target capture kits, and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).
```

> TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).

Reviewer: comma

```suggestion
TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).
```
```suggestion
TCGA samples were captured using various WXS target capture kits, and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).
```

The manufacturers provided the input interval BED files for both panel and WXS data for PBTA samples.

> We padded all panel and WXS BED files were by 100 bp on each side for Strelka2, Mutect2, and VarDict runs and by 400 bp for the Lancet run.  

Reviewer: extra word "were" and add "and" for parallel structure
```suggestion
We padded all panel and WXS BED files by 100 bp on each side for the Strelka2, Mutect2, and VarDict runs and by 400 bp for the Lancet run.  
```

> We padded all panel and WXS BED files were by 100 bp on each side for Strelka2, Mutect2, and VarDict runs and by 400 bp for the Lancet run.  

Reviewer: extra word
```suggestion
We padded all panel and WXS BED files by 100 bp on each side for the Strelka2, Mutect2, and VarDict runs and by 400 bp for the Lancet run.  
```

For WGS calling, we utilized the non-padded BROAD Institute interval calling list [`wgs_calling_regions.hg38.interval_list`](https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list), comprised of the full genome minus N bases, unless otherwise noted below.

> We ran Strelka2 [@doi:10/gdwrp4] v2.9.3  using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final Strelka2 VCF for PASS variants.

Reviewer: extra space
```suggestion
We ran Strelka2 [@doi:10/gdwrp4] v2.9.3 using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final Strelka2 VCF for PASS variants.
```

> We ran Strelka2 [@doi:10/gdwrp4] v2.9.3  using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final Strelka2 VCF for PASS variants.

Reviewer: extra space
```suggestion
We ran Strelka2 [@doi:10/gdwrp4] v2.9.3 using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final Strelka2 VCF for PASS variants.
```

We ran Mutect2 from GATK v4.1.1.0 according to Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl], and we filtered the final Mutect2 VCF for PASS variants.
To manage memory issues, we ran VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] using 20 Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.

> Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained.

Reviewer: Note - we used VAF in this section earlier, but here the acronym is defined. Acronym definition should occur at first usage in section,

> Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained.

Reviewer: Note - we used VAF in this section earlier, but here the acronym is defined. Acronym definition should occur at first usage in section,

SNV and INDEL hotspots from v2 of MSKCC's database [@url:https://www.cancerhotspots.org/#/download] plus the C228T and C250T TERT promoter mutations [@doi:10.3390/ijms21176034] were annotated.
SNVs were annotated by matching amino acid position (`Protein_position` column in MAF file) with SNVs in the MSKCC database, splice sites were matched to `HGVSp_Short`values in the MSKCC database, and INDELs were matched based on amino acid present within the range of INDEL hotspots values in the MSKCC database.
Non-hotspot annotated variants with a normal depth of <= 7 and/or gnomAD AF > 0.001 were removed as potential germline variants.

> TERT promoter mutations were matched using hg38 coordinates from Zvereva, et. al [@doi:10.3390/ijms21176034]: C228T occurs at 5:1295113, is annotated as existing variant `s1242535815`, `COSM1716563`, `COSM1716558`, and is 66 bp away from TSS and C250T occurs at Chr5:1295135, is annotated as existing variant `COSM1716559`, and is 88 bp away from TSS.

Author: updated:
```
We matched TERT promoter mutations using hg38 coordinates from Zvereva, et. al [@doi:10.3390/ijms21176034]: C228T occurs at 5:1295113, is annotated as existing variant `s1242535815`, `COSM1716563`, `COSM1716558`, and is 66 bp away from TSS and C250T occurs at Chr5:1295135, is annotated as existing variant `COSM1716559`, and is 88 bp away from TSS.
```

There is a small proportion of gene symbols that map to multiple Ensembl gene identifiers (in GENCODE v27, 212 gene symbols map to 1866 Ensembl gene identifiers), termed multi-mapped gene symbols.

We first removed genes with no expression from the RSEM abundance data by requiring an FPKM > 0 in at least 1 sample across the PBTA cohort.
We computed the mean FPKM across all samples per gene. 
For each multi-mapped gene symbol, we chose the Ensembl identifier corresponding to the maximum mean FPKM with the goal of choosing the identifier that best represented the expression of the gene.
After collapsing gene identifiers, there were a total of 46,400 uniquely expressed genes in the poly-A dataset and a total of 53,011 uniquely expressed genes remaining in the stranded dataset.
Additional details can be found in the [`collapse-rnaseq` analysis module](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/collapse-rnaseq).

##### Gene fusion detection

> We set up Arriba v1.1.0 [@doi:10.1101/gr.257246.119] and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.

Reviewer: `v` or no `v` for both versions (unless versions themselves really have diff names like that!)

> We set up Arriba v1.1.0 [@doi:10.1101/gr.257246.119] and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.

Reviewer: `v` or no `v` for both versions (unless versions themselves really have diff names like that!)

For Arriba, we used a blacklist file `blacklist_hg38_GRCh38_2018-11-04.tsv.gz` from the Arriba release to remove recurrent fusion artifacts and transcripts present in healthy tissue.
We provided Arriba with strandedness information for stranded samples, or we set it to auto-detection for poly-A samples.
We used [FusionAnnotator](https://github.com/FusionAnnotator/FusionAnnotator) on Arriba fusion calls to harmonize annotations with those of STAR-Fusion.
The RNA expression and fusion workflows can be found in the [D3b GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_RNAseq_workflow.cwl).
The FusionAnnotator workflow can also be found in the [D3b GitHub repository](https://github.com/d3b-center/FusionAnnotator).

### QUANTIFICATION AND STATISTICAL ANALYSIS

##### Recurrently mutated genes and co-occurrence of gene mutations

Using the consensus SNV calls, we identified genes that were recurrently mutated in the OpenPBTA cohort, including nonsynonymous mutations with a VAF > 5% among the set of independent samples.

> The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in `maftools v. 2.2.10` [@doi:10.1101/gr.239244.118].

Reviewer: We haven't been using code style elsewhere. Also suggested  `v. ` --> `v`

```suggestion
The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in maftools v2.2.10 [@doi:10.1101/gr.239244.118].
```

> The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in `maftools v. 2.2.10` [@doi:10.1101/gr.239244.118].

Reviewer: We haven't been using code style elsewhere. Also suggested  `v. ` --> `v`, your call

```suggestion
The set of nonsynonymous mutations was determined using ENSEMBL Variant Effect Predictor [@doi:10/gdz75c] annotations, including "High" and "Moderate" consequence types as defined in maftools v2.2.10 [@doi:10.1101/gr.239244.118].
```

##### Focal Copy Number Calling

We added the ploidy inferred via Control-FREEC to the consensus CNV segmentation file and used the ploidy and copy number values to define gain and loss values broadly at the chromosome level.

> We used `bedtools coverage` [@url:https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html; @doi:10.1093/bioinformatics/btq033] to add cytoband status using the (UCSC cytoband file)[url:http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz] [@doi:10.1093/nar/gks1048].

Reviewer: this is probably good in backticks since it's 2 words

> We used `bedtools coverage` [@url:https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html; @doi:10.1093/bioinformatics/btq033] to add cytoband status using the (UCSC cytoband file)[url:http://hgdownload.cse.ucsc.edu/goldenpath/hg38/database/cytoBand.txt.gz] [@doi:10.1093/nar/gks1048].

Reviewer: this is probably good in backticks since it's 2 words

> The output status call fractions, which are values of the loss, gain, and callable fractions of each cytoband region, were used to define dominant status at the cytoband-level.

Reviewer: ```suggestion
We used the output status call fractions, which are values of the loss, gain, and callable fractions of each cytoband region, to define dominant status at the cytoband-level.
```

> The output status call fractions, which are values of the loss, gain, and callable fractions of each cytoband region, were used to define dominant status at the cytoband-level.

Reviewer: ```suggestion
We used the output status call fractions, which are values of the loss, gain, and callable fractions of each cytoband region, to define dominant status at the cytoband-level.
```

We calculated the weighted means of each status call fraction using band length.
We used the weighted means to define the dominant status at the chromosome arm-level.

A status was considered dominant if more than half of the region was callable and the status call fraction was greater than 0.9 for that region.
We adopted a 0.9 threshold to ensure that the dominant status fraction call is greater than the remaining status fraction calls in a region.

We aimed to define focal copy number units to avoid calling adjacent genes in the same cytoband or arm as copy number losses or gains where it would be more appropriate to call the broader region a loss or gain.
To determine the most focal units, we first considered the dominant status calls at the chromosome arm-level.
If the chromosome arm dominant status was callable but not clearly defined as a gain or loss, we instead included the cytoband-level status call.
Similarly, if a cytoband dominant status call was callable but not clearly defined as a gain or loss, we instead included gene-level status call.

> To obtain the gene-level data, we used the `mergeByOverlaps` [function](@url:https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the IRanges package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the [GENCODE v27 annotation file](https://www.gencodegenes.org/human/release_27.html).

Reviewer: same `backticks` comment

> To obtain the gene-level data, we used the `mergeByOverlaps` [function](@url:https://www.rdocumentation.org/packages/IRanges/versions/2.6.1/topics/findOverlaps-methods) from the IRanges package in R [@doi:10.1371/journal.pcbi.1003118] to find overlaps between the segments in the consensus CNV file and the exons in the [GENCODE v27 annotation file](https://www.gencodegenes.org/human/release_27.html).

Reviewer: same `backticks` comment

#### Survival analysis

Overall survival, denoted `OS_days`, was calculated as days since initial diagnosis.
Progression-free survival, denoted `PFS_days`, was calculated as days since last event.

#### Prediction of participants' genetic sex

Participant metadata includes a reported gender.
We used WGS germline data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.
This analysis may also reveal samples that may have been contaminated.
We used the `idxstats` utility from SAMtools [@pmid:19505943] to calculate read lengths, the number of mapped reads, and the corresponding chromosomal location for reads to the X and Y chromosomes.
We used the fraction of total normalized X and Y chromosome reads that were attributed to the Y chromosome as a summary statistic.
We reviewed this statistic in the context of reported gender and determined that a threshold of less than 0.2 clearly delineated female samples.

> Fractions greater than 0.4 were predicted to be males.

Author: updated next commit, extra ``` here

#### Survival analysis

Overall survival, denoted `OS_days`, was calculated as days since initial diagnosis.
Progression-free survival, denoted `PFS_days`, was calculated as days since last event.

#### Prediction of participants' genetic sex

> Participant metadata includes a reported gender.

Reviewer: included? Not sure about tense here actually. Your call

> Participant metadata includes a reported gender.

Reviewer: included? Not sure about tense here actually. Your call

We used WGS germline data, in concert with the reported gender, to predict participant genetic sex so that we could identify sexually dimorphic outcomes.

> This analysis may also reveal samples that may have been contaminated.

Reviewer: "may have also revealed" I think is a better tense here?

> This analysis may also reveal samples that may have been contaminated.

Reviewer: "may have also revealed" I think is a better tense here?
